myst_nbs/variational_inference/bayesian_neural_network_advi.myst.md

```diff
@@ -378,35 +321,14 @@ For fun, we can also look at the trace. The point is that we also get uncertaint
 az.plot_trace(trace);
 ```
 
-## Summary
-
-Hopefully this blog post demonstrated a very powerful new inference algorithm available in PyMC3: {class}`~pymc3.variational.inference.ADVI`. I also think bridging the gap between Probabilistic Programming and Deep Learning can open up many new avenues for innovation in this space, as discussed above. Specifically, a hierarchical neural network sounds pretty bad-ass. These are really exciting times.
-
-## Next steps
-
-[`Theano`](https://github.com/Theano/Theano), which is used by `PyMC3` as its computational backend, was mainly developed for estimating neural networks and there are great libraries like [`Lasagne`](https://github.com/Lasagne/Lasagne) that build on top of `Theano` to make construction of the most common neural network architectures easy. Ideally, we wouldn't have to build the models by hand as I did above, but use the convenient syntax of `Lasagne` to construct the architecture, define our priors, and run ADVI. 
-
-You can also run this example on the GPU by setting `device = gpu` and `floatX = float32` in your `.theanorc`.
-
-You might also argue that the above network isn't really deep, but note that we could easily extend it to have more layers, including convolutional ones to train on more challenging data sets.
-
-I also presented some of this work at PyData London, view the video below:
-<iframe width="560" height="315" src="https://www.youtube.com/embed/LlzVlqVzeD8" frameborder="0" allowfullscreen></iframe>
-
-Finally, you can download this NB [here](https://github.com/twiecki/WhileMyMCMCGentlySamples/blob/master/content/downloads/notebooks/bayesian_neural_network.ipynb). Leave a comment below, and [follow me on twitter](https://twitter.com/twiecki).
+You might argue that the above network isn't really deep, but note that we could easily extend it to have more layers, including convolutional ones to train on more challenging data sets.
 
 ## Acknowledgements
 
 [Taku Yoshioka](https://github.com/taku-y) did a lot of work on ADVI in PyMC3, including the mini-batch implementation as well as the sampling from the variational posterior. I'd also like to the thank the Stan guys (specifically Alp Kucukelbir and Daniel Lee) for deriving ADVI and teaching us about it. Thanks also to Chris Fonnesbeck, Andrew Campbell, Taku Yoshioka, and Peadar Coyle for useful comments on an earlier draft.
 
 +++
 
```

Comment:
add authors section. This allows to keep the link to the original blog post (even if updated multiple times and it becomes harder to recognize the origin) as well as listing authors with their names as clickable links to their github profile, twitter...